Makes russian work properly

SireTurret2 opened the pull request:
РАЗ РАЗ РАЗ ЭТО ХАРДБАСС

SireTurret2 committed:
Makes russian work properly.

SireTurret2 commented:
@mattroks101

SireTurret2 commented:
![Screenshot_19](https://user-images.githubusercontent.com/65874511/92780528-f8f53c80-f3ab-11ea-9389-4fb35dc7a2c9.png)
![Screenshot_20](https://user-images.githubusercontent.com/65874511/92780536-fabf0000-f3ab-11ea-8d66-a8b3108cafb7.png)

mattroks101 commented:
Does this break normal english?

mattmatterson111 commented on code/_helpers/text.dm:
```diff
@@ -207,6 +207,15 @@
 		t = "[t] "
 	return t
 
+/proc/add_characters(c, n = 0) //Adds whatever character in 'c' and repeats that process 'n' times then returns text string as "XXXXXXXXXX"
```
I don't see this proc being called anywhere. Am I missing something?

SireTurret2 replied:
Removed it. It was meant to be used for nuclear codes.

mattmatterson111 commented on code/_helpers/text.dm:
```diff
@@ -227,7 +236,7 @@
 //Returns a string with the first element of the string capitalized.
 /proc/capitalize(var/t as text)
-	return uppertext(copytext(t, 1, 2)) + copytext(t, 2)
+	return uppertext(copytext_char(t, 1, 2)) + copytext_char(t, 2)
```
This looks like the only thing that's actually doing anything, what's that other thing for?

SireTurret2 committed:
Removes the unused code piece

It was supposed to be for nuke codes originally, but that doesn't exist in IS12 Warfare yet.
